WordPress/playground-tools
# Remove `playground-website` from the commands

- Related to https://github.com/WordPress/wordpress-playground/issues/435#issuecomment-1765745642

## What?



Removes the commands related to `playground-website` from the package.
It removes the expectation that it will spin-up a project that doesn't live in this repo.

Another alternative would be starting `wp-now` command 🤔 .

## Why?



They are leftovers from the previous repo. And that package lives on https://github.com/WordPress/wordpress-playground/tree/trunk/packages/playground/website

## Testing Instructions





- Run `npm run dev``
- Observe `Missing script: "dev"`
- Run `npm run preview`
- Observe Missing script: "preview"`
